anta/tests/vxlan.py

```diff
@@ -5,19 +5,26 @@
 from anta.models import AntaCommand, AntaTest
 
 
-class VerifyVxlan(AntaTest):
+class VerifyVxlan1Interface(AntaTest):
```

Comment:
if we change the name we probably need to write some kind of note somewhere ... in the docs ?

Reply:
Agreed! Done.